### ISSUE TITLE
softmaxloss vs cross entropy

### ISSUE THREAD

**Issue by mithunpaul08** [UKPLab/sentence-transformers#242]
Just wanted to clarify one thing: are you calling cross entropy loss as softmax loss? Why am asking is I didn't see any `torch.nn.softmax(`) in the file `losses/SoftmaxLoss.py`. Its highly possible that my understanding is wrong, but when I learned from good fellow's book, they say that Softmax (the sigmoid general function usually used in the last layer) is different than Crossentropy (a loss or cost function
![Screen Shot 2020-05-21 at 1 08 05 AM](https://user-images.githubusercontent.com/1056029/82538055-fc646f80-9aff-11ea-8f09-f53c3104ebd4.png)
).

**Comment by nreimers** [comment]
The class uses CrossEntropyLoss from Pytorch. Have a look here, it already incoperates softmax with negative log likelihood in it:
https://pytorch.org/docs/stable/nn.html#crossentropyloss
